tensornetwork/backends/jax/jax_backend.py

```diff
@@ -25,6 +25,13 @@
 
 _CACHED_MATVECS = {}
 _CACHED_FUNCTIONS = {}
+_MIN_RES_THRESHS = {
+    np.dtype(np.float16): 1E-3,
+    np.dtype(np.float32): 1E-6,
+    np.dtype(np.float64): 1E-12,
+    np.dtype(np.complex128): 1E-12,
```

Comment:
Nice way of doing this

Reply:
An alternative (but this is fine) would be to make it proportional to the machine epsilon of the given data type: 1000 * np.finfo(dtype).eps or something

Reply:
that's more elegant, I agree. Can we use the same prefactor in all cases? for float16 we would have res_thres ~1